inc/menu.class.php

```diff
@@ -329,7 +329,7 @@ static function displayMenu($type = "big") {
       }
 
       if (Session::haveRight('plugin_fusioninventory_ignoredimportdevice', READ)) {
-         $a_menu[2]['name'] = __('Ignored import devices', 'fusioninventory');
+         $a_menu[2]['name'] = __('Equipment ignored on import', 'fusioninventory');
```

Comment:
I would say "Asset skipped during the import"

Reply:
Ok